src/components/accessories/feedback/utils.ts

```diff
@@ -0,0 +1,13 @@
+import * as params from "../../../customization/feedback.json";
+import { IFeedbackProps } from "./type";
+
+const collectorUrl =
+  "https://openhospital.atlassian.net/s/d41d8cd98f00b204e9800998ecf8427e-T/r5gghz/b/3/bc54840da492f9ca037209037ef0522a/_/download/batch/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector/com.atlassian.jira.collector.plugin.jira-issue-collector-plugin:issuecollector.js?locale=en-US&collectorId=53f7755c";
+
+export const feedbackProps: IFeedbackProps = {
+  enabled: params?.enabled ?? true,
+  url: params?.url ?? collectorUrl,
+  email: params?.email ?? "example@mail.com",
+  version: params?.version ?? "0.0.1",
+  priority: params?.priority ?? "2",
+};
```

Comment:
When you import a **JSON** file, you don't need to remap all the properties into a javascript object for it is already parsed. You could simply use that as the following:

```
import feedback from "../../../customization/feedback.json";

console.log(feedback.enabled); 
```

Moreover, the default values should be given by a default **JSON** file that merges the one carrying the customs. The `??` operator should be avoided in this case.

e.g.:
```
// @file /customization/feedback/index.js

import defaults from "./defaults.json";
import customs from "./customs.json";

export const feedback = { ...defaults, ...customs }
```